apps/chef_db/priv/pgsql_statements.config

```diff
@@ -379,3 +379,9 @@
    WHERE sc.org_id = $1
      AND sc.sandbox_id = $2
      AND c.checksum IS NULL">>}.
+
+{list_keys_for_actor, <<"SELECT key_name, (expires_at <= CURRENT_TIMESTAMP at time zone 'utc')
+                          FROM keys WHERE id= $1">>}.
+{find_key_by_id_and_name, <<"SELECT id, key_name, public_key, key_version, created_at, expires_at
+                               FROM keys
+                              WHERE (id = $1 AND name = $2) LIMIT 1">>}.
```

Comment:
Yes. Fortunately we're not using this query yet (or it would've exploded). Will update.